Meadowlark-js/lerna.json

```diff
@@ -3,12 +3,7 @@
   "packages": [
     "packages/*"
   ],
-  "version": "v0.3.0-pre-35",
+  "version": "0.4.0-pre.0",
```

Comment:
This update had to be done manually for the first version, this will be done automatically in future changes